src/transformers/models/camembert/modeling_camembert.py

```diff
@@ -728,7 +728,11 @@ def forward(self, features, **kwargs):
 
     def _tie_weights(self):
         # To tie those two weights if they get disconnected (on TPU or when the bias is resized)
-        self.bias = self.decoder.bias
+        # For accelerate compatibility and to not break backward compatibility
+        if self.decoder.bias.device == torch.device("meta"):
```

Comment:
The test will probably break if PyTorch is < 1.9, so we need a safer way to test if the device is meta (can be in an util if the test ends up being long).

Reply:
I propose a fix here, 05da6938bc15733d739a995dd9df4398f3301826 
I am not sure if `device.type` can be retrieved for PT<1.9 but it is something that I have seen on `accelerate` I think

Reply:
Here is a quick try on torch == 1.7.1 ! 
```
>>> torch.__version__
'1.7.1'
>>> vec = torch.randn(1, 1)
>>> vec.device
device(type='cpu')
>>> vec.device.type
'cpu'
```

Reply:
Nice!